experimental/addons/kafka-operator/README.md

```diff
@@ -0,0 +1,237 @@
+$ curl http://localhost:8080
+{"bridge_version":"0.24.0"}
+```
+
+**Create kafka-rebalance to depict every partition of topic**
+
+Apply below YAML to create kafka-connector in `kafka-operator` namespace:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: kafka-rebalance-sample
+spec:
+  components:
+    - type: kafka-rebalance
+      name: kafka-rebalance
+      properties:
+        goals:
+        - CpuCapacityGoal
+        - NetworkInboundCapacityGoal
+        - DiskCapacityGoal
+        - RackAwareGoal
+        - MinTopicLeadersPerBrokerGoal
+        - NetworkOutboundCapacityGoal
+        - ReplicaCapacityGoal
+```
+
+Now, Verify.
+
+```shell
+NAME              CLUSTER         PENDINGPROPOSAL   PROPOSALREADY   REBALANCING   READY   NOTREADY
+kafka-rebalance   kafka-cluster                                                   True        
+```
+
+### External datasource connection to kafka broker
+
+**Create kafka-connect cluster**
+
+Apply below YAML to create kafka-connect in `kafka-operator` namespace:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: kafka-connect-sample
+spec:
+  components:
+    - type: kafka-connect
+      name: kafka-connect
+      properties:
+        version: 3.3.2
+        replicas: 1
+        bootstrapServers: "kafka-cluster-kafka-bootstrap:9093"
+        tls:
+          trustedCertificates:
+            - secretName: kafka-cluster-cluster-ca-cert
+              certificate: ca.crt
+        config:
+          group.id: connect-cluster
+          offset.storage.topic: connect-cluster-offsets
+          config.storage.topic: connect-cluster-configs
+          status.storage.topic: connect-cluster-status
+          config.storage.replication.factor: -1
+          offset.storage.replication.factor: -1
+          status.storage.replication.factor: -1
+```
+
+Now, verify.
+
+```shell
+$ kubectl get kafkaConnect -n kafka-operator
+NAME            DESIRED REPLICAS   READY
+kafka-connect   1                  True
+```
+
+**Create kafka-connector to connect with `kafka-connect` cluster and external datasource**
+
+Apply below YAML to create kafka-connector in `kafka-operator` namespace:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: kafka-connector-sample
+spec:
+  components:
+    - type: kafka-connector
+      name: kafka-connector
+      properties:
+        class: org.apache.kafka.connect.file.FileStreamSourceConnector
+        tasksMax: 1
+        config:
+          file: /opt/kafka/LICENSE
+          topic: kafka-topic
+```
+
+Now, Verify.
+
+```shell
+$ kubectl get kafkaConnector -n kafka-operator
+NAME              CLUSTER         CONNECTOR CLASS                                           MAX TASKS   READY
+kafka-connector   kafka-connect   org.apache.kafka.connect.file.FileStreamSourceConnector   1           True
+```
+
+Enjoy your Apache Kafka cluster, running on Minikube!.
+For more visit on https://strimzi.io/.
```

Comment:
The issue here is the same with the rocket-mq addon, how will the real world user use the kafka addon? Maybe we need some example code as "business logic" to use the topic/instace provided here.